test/AssemblySet_basic_test.py

```diff
@@ -173,33 +155,13 @@ def test_get_assembly_set(
 
     if params.get(REF_PATH_TO_SET, []) != []:
         # add in a value for the REF_PATH_TO_SET
-        params[REF_PATH_TO_SET] = [assembly_set_ref]
+        params[REF_PATH_TO_SET] = [set_ref]
 
-    fetched_set = set_api_client.get_reads_alignment_set_v1(context, params)[0]
+    fetched_set = set_api_client.get_assembly_set_v1(context, params)[0]
```

Comment:
OOPS! This is where I lost any faith I had in humankind and realised that the `get_set` routines DGAF about what type of set they return.

Reply:
🤦‍♂️